packaging/torchaudio/meta.yaml

```diff
@@ -13,16 +13,16 @@ requirements:
   host:
     - python
     - setuptools
-{{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
+    - cpuonly
+    {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
 
   run:
     - python
     - typing
-{{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
+    {{ environ.get('CONDA_PYTORCH_CONSTRAINT') }}
 
 build:
-  number: {{ environ.get('TORCHAUDIO_BUILD_NUMBER') }}
-  string: py{{py}}_{{environ.get('TORCHAUDIO_BUILD_NUMBER')}}
+  string: py{{py}}
```

Comment:
you dont have a BUILD_NUMBER, are you sure you will never botch a binary build ever?

Reply:
My thinking is, if I botch build of `1.2.0`, then I will next build `1.2.0.post2` (because I don't get build number in pip wheels). Now the version has changed and I don't need it. But I could be wrong!